### DISCUSSION TITLE
Use a blur effect with i3-lock

### DISCUSSION THREAD

**Discussion by Feelsonic** [i3/i3#6247] [Q&A]
Hi,

I want to use a blur effect on my desktop when i3lock start to lock my screen.
I saw lot of way to make my wish. But i want the lightest way to make this.

Thank u.

**Comment by jpggithub** [comment]
Use a script like this:

```
#!/bin/sh

# future screenshot pixelated
img=/tmp/i3lock.png

# take screenshot
import -window root $img

# pixelate the screenshot
convert $img -scale 10% -scale 1000% $img

# launch i3lock with the pixelated screenshot
i3lock -i $img
```

**Comment by Pinchhh** [comment]
You can just download i3lock-fancy by running
`sudo apt install i3lock-fancy`
in terminal

then run i3lock-fancy from terminal or set a keybinding to it in your config file
this one is like the default i3lock but it blurs the backround instead of being a solid colour

:)
